src/QsCompiler/CompilationManager/CompilationUnit.cs

```diff
@@ -395,8 +395,9 @@ internal void UpdateCallables(IEnumerable<QsCallable> updates)
                     if (header.Kind.IsTypeConstructor) 
                     {
                         var specLocation = new QsLocation(header.Position, header.SymbolRange);
-                        var defaultSpec = new QsSpecialization(QsSpecializationKind.QsBody, header.QualifiedName, header.Attributes, header.SourceFile, specLocation, 
-                            QsNullable<ImmutableArray<ResolvedType>>.Null, header.Signature, SpecializationImplementation.Intrinsic, ImmutableArray<string>.Empty, QsComments.Empty);
+                        var defaultSpec = new QsSpecialization(QsSpecializationKind.QsBody, header.QualifiedName, header.Attributes, CapabilityLevel.Unset, 
```

Comment:
I'm not sure this will work; the capability level was moved from QsSpecialization into the inferred data of the specialization signature. I think this is dangling from a WIP commit that moved the field but didn't update this constructor (or several others).
```suggestion
                        var defaultSpec = new QsSpecialization(QsSpecializationKind.QsBody, header.QualifiedName, header.Attributes, 
```